.travis.yml

```diff
@@ -132,97 +172,23 @@ jobs:
       python:
         - "3.7"
       env:
-        - CONTAINERD_VERSION=1.2.6
-        - KUBECONFIG="$HOME/.kube/config"
-        - KUBERNETES_VERSION=v1.15.3
-        - MINIKUBE_VERSION=v1.3.1
-        - CHANGE_MINIKUBE_NONE_USER=true
-        - MINIKUBE_WANTUPDATENOTIFICATION=false
-        - MINIKUBE_WANTREPORTERRORPROMPT=false
-        - MINIKUBE_HOME=$HOME
-
-      before_install:
-      # See: https://github.com/containerd/cri/blob/master/docs/installation.md
-      - sudo apt-get update && sudo apt-get install libseccomp2 -y
-      - curl --fail -sLO "https://storage.googleapis.com/cri-containerd-release/cri-containerd-${CONTAINERD_VERSION}.linux-amd64.tar.gz"
-      - sudo tar --no-overwrite-dir -C / -xzf cri-containerd-${CONTAINERD_VERSION}.linux-amd64.tar.gz
+        - KUBERNETES_VERSION=v1.17.2
+        - CALICO_VERSION=3.8
 
       before_script:
       # install dependent binaries
       - curl --fail -Lo kubectl "https://storage.googleapis.com/kubernetes-release/release/${KUBERNETES_VERSION}/bin/linux/amd64/kubectl" && chmod +x kubectl && sudo mv kubectl /usr/local/bin/
-      - curl --fail -Lo minikube "https://storage.googleapis.com/minikube/releases/${MINIKUBE_VERSION}/minikube-linux-amd64" && chmod +x minikube && sudo mv minikube /usr/local/bin/
-      - mkdir -p $HOME/.kube $HOME/.minikube
-      - touch $KUBECONFIG
-      - minikube config set vm-driver none
+      - curl -Lo ./kind https://github.com/kubernetes-sigs/kind/releases/download/v0.7.0/kind-$(uname)-amd64
+      - chmod +x ./kind
+      - sudo mv ./kind /usr/local/bin/kind
```

Comment:
We still need here the local registry: https://kind.sigs.k8s.io/docs/user/local-registry/ (since the secure env vars not available in other repos)